src/Gui/DlgParameterImp.cpp

```diff
@@ -117,6 +117,21 @@ DlgParameterImp::DlgParameterImp( QWidget* parent,  Qt::WindowFlags fl )
     connect(paramGroup, SIGNAL(currentItemChanged(QTreeWidgetItem*, QTreeWidgetItem*)), 
             this, SLOT(onGroupSelected(QTreeWidgetItem*)));
     onGroupSelected(paramGroup->currentItem());
+
+    connect(ui->findGroupLE, SIGNAL(textChanged(QString)),
+        this, SLOT(on_findGroup_changed(QString)));
+
+    // setup for function on_findGroup_changed:
+    // store the current font properties because
+    // we don't know what style sheet the user uses for FC

```

Comment:
`s/style sheet/stylesheet/`

Reply:
What do you mean?, "style sheet" is the right term: https://en.wikipedia.org/wiki/Style_sheet

Reply:
@donovaly I guess both are used interchangeably in the FC code. Sorry for the noise.